packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -62,3 +65,49 @@ export function getArrayWithoutDuplicates(array: Array<Coord>): Array<Coord> {
 
     return returnArray;
 }
+
+export function getSlopeStringForLine(line: PairOfPoints, strings): string {
+    const slope = (line[1][1] - line[0][1]) / (line[1][0] - line[0][0]);
+    if (slope === Infinity || slope === -Infinity) {
```

Comment:
FWIW, you could use `Number.isFinite` here:

```suggestion
    if (!Number.isFinite(slope)) {
```

That will also cause `srLinearGraphSlopeVertical` to be returned if the slope is `NaN`. (A slope of `NaN` shouldn't be possible, since we prevent the two control points of the line from having the same coordinates, so this is more paranoia than anything.)